app/exceptions/handlers.py

```
@@ -0,0 +1,85 @@
+		f"[UNHANDLED_EXCEPTION] {exc_type} occurred | Path: {path} | Method: {method} | Status: {status_code}"
+	)
+	
+	return APIResponse.error(
+		message=app.exceptions.exception_constants.SERVICE_UNAVAILABLE,
+		status_code=status_code,
+		debug={"exception": exc_type, "str": str(exc)}
+	)
+
+
+def devdox_base_exception_handler(request: Request, exc: DevDoxAPIException):
+	path = request.url.path
```

Pylint (3.3.7)

[warning] Bad indentation. Found 1 spaces, expected 4

(W0311)